### ISSUE TITLE
23.07.13(목) Issue

### ISSUE THREAD

**Issue by SaFD-00** [SaFD-00/2023_RCV_SUMMER_URP#12]
# <공부한 내용>
## SSD 코드
- 체크포인트 저장 (https://honeyjamtech.tistory.com/39)
- 절대경로 구하기: os.path.abspath()
- XML 파싱(https://wikidocs.net/21140)
- x.extend(y): 리스트 x 끝에 가장 바깥쪽 iterable한 리스트 y의 모든 항목을 넣습니다.
- iter / iterable / iterator
- torch.cat: 원하는 dimension 방향으로 텐서를 나란하게 쌓아줍니다.
- Tensor 타입 지정(https://dk-kang.tistory.com/entry/PyTorch-%EC%8B%A4%EC%8A%B5-Tensor-%EC%A1%B0%EC%9E%91%ED%95%98%EA%B8%B01)
- torch.unsqueeze(input, dim): dim에 1인 차원을 생성하는 함수
- torch.clamp(input, min=None, max=None, *, out=None): torch.min을 min으로, torch.max를 max로 정규화 (out = output tensor)
